spec/typed_parameters/comparison_spec.rb

```diff
@@ -25,10 +25,25 @@
       end
 
       context "when the request body does not adhere to the rules format" do
-        let(:name) { 1_000 }
+        context "when a non-specified parameter is included in the request body" do
+          let(:params) { { name: "M@", email: "matt@zipmark.com" } }
```

Comment:
The two examples below represent (1) ignoring them, not adding an error but filtering out the non-permitted param, or (2) adding an error as we do now in OTX.